Industrial CI from ROS main failing consistently

[On this PR](https://github.com/ros-controls/ros2_control/pull/136) we had to disable "Industrial CI from ROS main" due to consistent failure none of us were able to reproduce with (supposedly) identical installations.



The error was

```
  Starting function 'build_target_workspace'
  Starting >>> hardware_interface
  --- stderr: hardware_interface
  In file included from /opt/ros/foxy/include/control_msgs/msg/dynamic_joint_state.hpp:9,
                   from /root/target_ws/src/ros2_control/hardware_interface/include/hardware_interface/robot_hardware.hpp:22,
                   from /root/target_ws/src/ros2_control/hardware_interface/src/robot_hardware.cpp:15:
  /opt/ros/foxy/include/control_msgs/msg/detail/dynamic_joint_state__traits.hpp:27:25: error: expected initializer before '<' token
     27 | inline const char * name<control_msgs::msg::DynamicJointState>()
        |                         ^
  make[2]: *** [CMakeFiles/hardware_interface.dir/build.make:115: CMakeFiles/hardware_interface.dir/src/robot_hardware.cpp.o] Error 1
  make[1]: *** [CMakeFiles/Makefile2:166: CMakeFiles/hardware_interface.dir/all] Error 2
  make[1]: *** Waiting for unfinished jobs....
  make: *** [Makefile:141: all] Error 2
  ---
  Failed   <<< hardware_interface [9.54s, exited with code 2]
```

@Karsten1987 @piraka9011 do you guys know by any chance what this error may originate from?

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -29,7 +29,7 @@ jobs:
           - {ROS_DISTRO: foxy, ROS_REPO: testing}
     runs-on: ubuntu-latest
     steps:
-      - run: apt-get update -qq && apt-get upgrade
+      - run: sudo apt-get update -qq && sudo apt-get upgrade
       - uses: actions/checkout@v1
       - uses: 'ros-industrial/industrial_ci@master'
         env: ${{matrix.env}}

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -29,6 +29,7 @@ jobs:
           - {ROS_DISTRO: foxy, ROS_REPO: testing}
     runs-on: ubuntu-latest
     steps:
+      - run: apt-get update -qq && apt-get upgrade
       - uses: actions/checkout@v1
       - uses: 'ros-industrial/industrial_ci@master'
         env: ${{matrix.env}}

